src/styles/global.css

```
@@ -67,11 +67,34 @@
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     background-color: hsl(var(--background));
+    /* Cyber Aurora handled by Background.svelte */
+    background-color: transparent;
+    background-attachment: fixed;
     color: hsl(var(--foreground));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)